scripts/create_master_checklist.py

```diff
@@ -211,10 +214,11 @@ def update_excel_file(input_excel_file, output_excel_file, checklist_data):
         # Next row
         row_counter += 1
 
+    number_of_checks = row_counter - row1

```

Comment:
moved this out if the "if args.verbose" block because it was causing an error if you execute the script without --verbose param